tests/test_test.py

```diff
@@ -100,9 +102,11 @@ def test_scan_for_tests_makecheck_am(self):
         content = 'check:'
         m_open = mock_open(read_data=content)
         with patch(self.open_name, m_open, create=True):
```

Comment:
Looks like make check is an autogenerated target for automake as far as I can tell so there isn't any need to test if the target exists as it can just be run and with no tests will report no errors.

Wouldn't I be best to remove 100-102 in that case though as it still checks if the file exists?

Reply:
Alrighty, verified and pushing an update.

Reply:
Okay, look good